### PR TITLE
[WM-1685] Add SonarCloud to CBAS-UI

### PR DESCRIPTION
**Pull request by kpierre13**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -93,3 +102,24 @@ jobs:
...
+        uses: actions/cache@v1
+        with:
+          path: ~/.sonar/cache
+          key: ${{ runner.os }}-sonar
+          restore-keys: ${{ runner.os }}-sonar
+
+      - name: SonarQube scan
+        uses: actions/checkout@v2
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```
Comment (salonishah11) [inline .github/workflows/publish.yml:124]: I don't think we have these tokens added to our repo? You might have to add them.

**Comment by salonishah11** [comment]
@kpierre13 I am not sure why but SonarCloud didn't add a comment to the PR with it's analysis (like it added for CBAS' PRs)

File: .github/workflows/publish.yml (hunk 1 above)
Comment (cjllanwarne) [reply .github/workflows/publish.yml:124]: +1. Ask in devops champions (and mention "atlantis")

**Comment by kpierre13** [comment]
@salonishah11 I saw this as well. I know in Terra-UI it doesn't pop up either, but I'll add that note when I reach out to devops champions.

**Review by salonishah11 — APPROVED** [review]

**Review by cjllanwarne — APPROVED** [review]

**Pull request merged**